README.md

```diff
@@ -277,6 +277,20 @@ SmartAPI is a set of REST-like APIs that expose many capabilities required to bu
 
 		String response = smartConnect.candleData(requestObejct);
 	}
+
+    /** Market Data */
+    public void getMarketData(SmartConnect smartConnect) {
+      
+        JSONObject payload = new JSONObject();
+        payload.put("mode", "FULL");
```

Comment:
There is 3 modes.    LTP("LTP"),
    OHLC("OHLC"),
    FULL("FULL");

why did you cover  FULL only? What about the rest?

Reply:
Apologies for the confusion. I demonstrated the "FULL" mode in the code snippet for simplicity. However, users can easily modify the code to request data in "LTP" or "OHLC" modes by changing the value within the payload JSON object. For "LTP" mode, they should replace the value with "LTP," and for "OHLC" mode, they should use "OHLC" instead.
For EG:
payload.put("mode", "FULL");
payload.put("mode", "LTP");
payload.put("mode", "OHLC");